Makefile

```diff
@@ -150,48 +157,60 @@ get-test-namespace: out/test-namespace
 .PHONY: e2e-setup
 e2e-setup: e2e-cleanup
 	$(Q)kubectl create namespace $(TEST_NAMESPACE)
+	$(Q)kubectl --namespace $(TEST_NAMESPACE) apply -f ./test/third-party-crds/postgresql_v1alpha1_database_crd.yaml
 
 .PHONY: e2e-cleanup
 e2e-cleanup: get-test-namespace
-	$(Q)-kubectl delete namespace $(TEST_NAMESPACE) --timeout=10s --wait
+	$(Q)-kubectl delete namespace $(TEST_NAMESPACE) --timeout=45s --wait
 
 .PHONY: test-e2e
 ## Runs the e2e tests locally from test/e2e dir
 test-e2e: e2e-setup
 	$(info Running E2E test: $@)
-	$(Q)GO111MODULE=on operator-sdk test local ./test/e2e --namespace $(TEST_NAMESPACE) --up-local --go-test-flags "-v -timeout=15m"
+	$(Q)operator-sdk --verbose test local ./test/e2e \
+			--debug \
+			--namespace $(TEST_NAMESPACE) \
+			--up-local \
+			--go-test-flags "-timeout=15m" || sleep 1800
 
 .PHONY: test-unit
 ## Runs the unit tests
 test-unit:
 	$(info Running unit test: $@)
-	$(Q)GO111MODULE=on GOCACHE=$(shell pwd)/out/gocache go test $(shell GOCACHE=$(shell pwd)/out/gocache go list ./...|grep -v e2e) -v -mod vendor $(TEST_EXTRA_ARGS)
+	$(Q)GO111MODULE=on GOCACHE=$(GOCACHE) go test $(shell GOCACHE="$(GOCACHE)" go list ./...|grep -v e2e) -v -mod vendor $(TEST_EXTRA_ARGS)
+
+.PHONY: test
+## Test: Runs unit and integration (e2e) tests
+test: test-unit test-e2e
 
 .PHONY: test-e2e-olm-ci
 test-e2e-olm-ci:
-	$(Q)sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:service-binding-operator-registry," ./test/e2e/catalog_source.yaml | kubectl apply -f -
-	$(Q)kubectl apply -f ./test/e2e/subscription.yaml
+	$(Q)sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:service-binding-operator-registry," ./test/operator-hub/catalog_source.yaml | kubectl apply -f -
+	$(Q)kubectl apply -f ./test/operator-hub/subscription.yaml
 	$(eval DEPLOYED_NAMESPACE := openshift-operators)
 	$(Q)./hack/check-crds.sh
 	$(Q)operator-sdk test local ./test/e2e --no-setup --go-test-flags "-v -timeout=15m"
```

Comment:
After merging Avni's PR, can you set `SERVICE_BINDING_OPERATOR_DISABLE_ELECTION` to `false` here?

Reply:
I can the PR has been merged already. Now making this change should make the test pass.

Reply:
If the environment variable is not set, the leader election will happen, right? Then no need to make any change here.

Reply:
Indeed, the environment variable `SERVICE_BINDING_OPERATOR_DISABLE_ELECTION` is not set on this target, therefore elections shall happen.